File: output/ChatGPT/avatar/Python/Java/codeforces_171_A.java
import java.util.Scanner;

public class codeforces_171_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String a = scanner.next();
        String b = scanner.next();
        int x = Math.max(a.length(), b.length());
        a = String.format("%" + x + "s", a).replace(' ', '0');
        b = String.format("%-" + x + "s", b).replace(' ', '0');
        StringBuilder s = new StringBuilder();
        int c = 0;
        for (int i = 0; i < x; i++) {
            int d = Character.getNumericValue(b.charAt(i)) + Character.getNumericValue(a.charAt(x - i - 1)) + c;
            if (d > 9) {
                s.append(Integer.toString(d).charAt(1));
                c = 1;
            } else {
                s.append(Integer.toString(d));
                c = 0;
            }
        }
        if (c == 1) {
            s.append('1');
        }
        System.out.println(Integer.parseInt(s.reverse().toString()));
    }
}

File: output/CodeLlama/avatar/Java/Python/codeforces_171_A.py
import sys
import math

def reverse(n):
    if n < 10:
        return n * 10
    t = n
    r = 0
    while t > 0:
        r = (r * 10) + t % 10
        t = t / 10
    return r

n = int(sys.stdin.readline())
m = int(sys.stdin.readline())
t = reverse(m)
print(t + n)